tests/testcases/core/libraries/plugin_api/EE_Register_Model_Extensions_Test.php

```diff
@@ -164,28 +164,28 @@ public function setUp(){
 	}
 
 
-
-	/**
-	 * OK's the creation of the esp_new_addon table, because this hooks in AFTER EE_UNitTestCase's callback on this same hook
-	 *
-	 * @param bool   $short_circuit
-	 * @param string $table_name
-	 * @param string $create_sql
-	 * @return array
-	 */
+    /**
+     * OK's the creation of the esp_new_addon table, because this hooks in AFTER EE_UNitTestCase's callback on this same hook
+     *
+     * @param bool   $short_circuit
+     * @param string $table_name
+     * @param string $create_sql
+     * @return bool
+     * @throws EE_Error
+     * @throws ReflectionException
+     */
 	public function dont_short_circuit_mock_table( $short_circuit = false, $table_name = '', $create_sql = '' ){
-		$table_analysis = EE_Registry::instance()->create( 'TableAnalysis', array(), true );
-		if( $table_name == 'esp_mock_attendee_meta' && ! $table_analysis->tableExists( $table_name) ){
-//			echo "\r\n\r\nDONT short circuit $sql";
+        $this->initTableAnalysis();
+		if( $table_name == 'esp_mock_attendee_meta' && ! $this->table_analysis->tableExists( $table_name) ){
```

Comment:
```suggestion
    $this->initTableAnalysis();
		if ($table_name == 'esp_mock_attendee_meta' && ! $this->table_analysis->tableExists($table_name)) {
```

Reply:
weird... my IDE should be formatting on paste AND on save, which obviously isn't happening...
also weird that his passed linting

good catch, will fix